src/faction.cpp

```diff
@@ -146,6 +146,8 @@ faction_template::faction_template( const JsonObject &jsobj )
     for( const JsonObject jao : jsobj.get_array( "epilogues" ) ) {
         epilogue_data.emplace( jao.get_int( "power_min", std::numeric_limits<int>::min() ),
                                jao.get_int( "power_max", std::numeric_limits<int>::max() ),
+                               jao.get_string( "dynamic",
+                                               "0000000" ), // old_guard, robofac, tacoma_commune, free_merchants, exodii, great_library, hells_raiders; 0 - anything; 1 - power >= 150; 2 - power < 150; 3,4,... - specific dynamic faction endings
```

Comment:
This will be hard to understand and maintain. You should probably make this an JSON array containing power requirement info of related factions. For example,

```javascript
"dynamic": [
  { "faction": "old_guard", "power_min": 150 },
  { "faction": "robofac", "power_max": 150 }
]
```